astropy/io/fits/scripts/fitscheck.py

```diff
@@ -106,6 +105,8 @@ def handle_options(args):
 
     if OPTIONS.checksum_kind == 'none':
         OPTIONS.checksum_kind = False
+    elif OPTIONS.checksum_kind == 'standard':
```

Comment:
This doesn't look related.

Reply:
Not really, just making sure that the option value makes sense, 'standard' evaluates as True but it has no special meaning and `True` should be used here.